frontend/server/src/Controllers/Run.php

```diff
@@ -571,8 +571,12 @@ public static function apiCreate(\OmegaUp\Request $r): array {
         }
 
         // Happy ending
-        $response['nextSubmissionTimestamp'] =
-            \OmegaUp\DAO\Runs::nextSubmissionTimestamp($contest);
+        $response['nextSubmissionTimestamp'] = \OmegaUp\DAO\Runs::nextSubmissionTimestamp(
+            $contest,
+            /*lastSubmissionTime=*/
+                new \OmegaUp\Timestamp(\OmegaUp\Time::get())
```

Comment:
```suggestion
            /*lastSubmissionTime=*/$submission->time
```

así se usa exactamente el mismo valor que se usó para insertar en la base de datos, y todo sería consistente!